### DISCUSSION TITLE
Fundamentals course Test Run lesson example errors

### DISCUSSION THREAD

**Discussion by bryanwrc** [firemodels/fds#12769] [Q&A] [answered]
I've just downloaded the latest Pyrosim (rev 2023.3.1312) and been going through the Pyrosim Fundamentals course with trying to replicate the first test run model. I get a REAC error. The first example is a mattress spring box fire using the SFPE WOOD-OAK as the reaction, and it creates a REAC_FUEL species, which is not in the FDS database. I get an warning/error when I try to run the simulation saying 'reduce molecular weight or define fuel specific heat' and it aborts. 

In the species properties for REAC_FUEL I switch from 'chemical formula' to 'molecular weight' and lower it from 29 to 28 g/mol. 

I get a ERROR: REAC .. with the following output
Starting FDS ...

 MPI Process      0 started on MSI

 Reading FDS input file ...

WARNING: SPEC REAC_FUEL is not in the table of pre-defined species. Any unassigned SPEC variables in the input were assigned the properties of nitrogen.

ERROR: REAC, Not enough carbon in the fuel for the specified CO_YIELD, SOOT_YIELD, and/or HCN_YIELD (CHID: tutorial_ex_t3)
forrtl: severe (157): Program Exception - access violation
Image              PC                Routine            Line        Source             
fds_openmp.exe     00007FF61316DA5E  Unknown               Unknown  Unknown
fds_openmp.exe     00007FF6130E16E7  Unknown               Unknown  Unknown
fds_openmp.exe     00007FF612D6F431  Unknown               Unknown  Unknown
fds_openmp.exe     00007FF612D2FEEE  Unknown               Unknown  Unknown
fds_openmp.exe     00007FF6136A5AA0  Unknown               Unknown  Unknown
KERNEL32.DLL       00007FFB4CBC257D  Unknown               Unknown  Unknown
ntdll.dll          00007FFB4E56AA58  Unknown               Unknown  Unknown
[runfds.exe] subprocess exit: 157

[tutorial ex t3.psm.zip](https://github.com/firemodels/fds/files/14924300/tutorial.ex.t3.psm.zip)

**Comment by rmcdermo** [comment]
You've uploaded your Pyrosim file.  On this forum we need the .fds file in order to help you.  You are also welcome to submit your question to the Pyrosim user support. 

https://support.thunderheadeng.com/pyrosim

**Reply by bryanwrc** [reply]
[tutorial_ex_t3 fds.zip](https://github.com/firemodels/fds/files/14925844/tutorial_ex_t3.fds.zip)

**Reply by rmcdermo** [reply]
That input file runs fine with the latest FDS.  You can download it [here](https://github.com/firemodels/fds/releases/tag/FDS-6.9.1).

What version of FDS is being print out in your command shell?

**Reply by bryanwrc** [reply]
It was FDS-6.8.0-0-g886e009-release, the release that came with the Pyrosim package. 

I downloaded the fds/smv package and now have the pyrosim point to it. It is using FDS-6.9.0-0-g6339569-release

I'm still getting Pyrosim errors, and ignored the warnings. Here is the fds file and output.

I don't see any smoke from this simulation. 

[tutorial_ex_1 fds out.zip](https://github.com/firemodels/fds/files/14931066/tutorial_ex_1.fds.out.zip)

**Reply by rmcdermo** [reply]
Something wrong with this file when I try to unzip it.  To upload, just copy your casename.fds file to casename.txt and upload it.  

But, first, the link I pointed you to is for FDS 6.9.1, which we just released yesterday.  Please try that, as it is what I am using to test, and it runs fine for me. 

Are you visualizing with Pryosim or Smokeview?  If Smokeview, then you need to load 3D Smoke (right click >> load >> 3D smoke).  If Pyrosim, then you need to contact Thunderhead at the link I gave above.

**Reply by bryanwrc** [reply, marked as answer]
I downloaded the newest version. 
I was able to visualize the results in pyrosim and SMV, I just needed to click on 'show' for 3D Smoke in pyrosim. 
This thread can be marked as resolved. Thank you.
[tutorial_ex_1.fds.txt](https://github.com/firemodels/fds/files/14931540/tutorial_ex_1.fds.txt)
[tutorial_ex_1 out.txt](https://github.com/firemodels/fds/files/14931541/tutorial_ex_1.out.txt)
